lib/ansible/modules/database/misc/elasticsearch_plugin.py

```diff
@@ -98,6 +98,12 @@
 - elasticsearch_plugin:
     state: absent
     name: mobz/elasticsearch-head
+    
```

Comment:
@Nitaco You have spaces in this blank line. I think that is the reason the CI tests are failing PEP8 rules :)